easybuild/easyconfigs/e/EMBOSS/EMBOSS-6.6.0_disable-embossupdate.patch

```diff
@@ -0,0 +1,22 @@
+remove build target embossupdate from Makefile
```

Comment:
@fizwit Can you add an extra line with either mentioning yourself as author, or mentioning a URL pointing to where this patch came from?

Reply:
Actually, this should mention me as author, since I added this patch ages ago (cfr. #963)

So, please add a line below this one:

```
author: Kenneth Hoste (HPC-UGent)

Reply:
fixed in 35dd3e25cb